examples/tsp/tsp.cc

```diff
@@ -89,7 +89,7 @@ auto main(int argc, char * argv[]) -> int
     auto succ = p.create_integer_variable_vector(n, 0_i, Integer(n - 1));
     auto dist = p.create_integer_variable_vector(n, 0_i, 745_i);
 
-    p.post(Circuit{succ, false, false});
```

Comment:
Hrm, wait... Have I been running experiments on the cluster with GAC turned on? For TSP we're trying to exactly mimic the propagation strength of MiniCP, which doesn't do GAC here.